test/test_hl_api.py

```diff
@@ -65,6 +65,8 @@ def test_parallel_population(population_params, genome_params, ea_params):
         fitness_per_n_processes.append(
             _test_population(population_params, genome_params, ea_params)
         )
+
+    assert fitness_per_n_processes[1] == pytest.approx(fitness_per_n_processes[2])
```

Comment:
please move this below the other asserts. first 0 & 1, then 0 & 2, then 1 & 2 should be checked.

Reply:
Can I also re-remove it? I only used it for finding the error in the test

Reply:
sure, you can also remove it. if 0 == 1 and 0 == 2, this implies 1 == 2 ;)